CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+#### Added  2024-04-26
```

Comment:
No date needed on these sections. Only used per release section header that the PR doesn't set.

Reply:
The section should be insert after the `## [Unreleased][]` section not before.

Reply:
removed date , section added after unreleased.